packages/cache/src/index.ts

```diff
@@ -0,0 +1,170 @@
+import { Redis } from 'ioredis';
+import { LRUCache } from 'lru-cache';
+import { logger } from '@prairielearn/logger';
+import * as Sentry from '@prairielearn/sentry';
+
+class Cache {
+  cacheEnabled = false;
+  cacheType = 'none';
+  memoryCache?: LRUCache<string, string>;
+  client?: Redis;
```

Comment:
Nit: consider naming this `redisClient` instead.

Reply:
Updated name to `redisClient` in [dacc892](https://github.com/PrairieLearn/PrairieLearn/pull/9360/commits/dacc892c32a53e09450e4b8ba921a9242e3bce7f).